Pipfile

```diff
@@ -6,31 +6,39 @@ name = "pypi"
 [packages]
 pyyaml = "~=5.4"
 types-PyYAML = "~=5.4"
-requests = "~=2.26"
+# TODO: The 'requests' package stays on 2.28 until we deprecate CentOS7.
+#       As newer version requires openssl1.1.1 where CentOS7 only provides openssl1.1.0.
+#       https://github.com/opensearch-project/opensearch-build/issues/3554
+requests = "<=2.28.1"
 types-requests = "~=2.25"
 pre-commit = "~=2.15.0"
 isort = "~=5.9"
 flake8 = "~=3.9"
-mypy = "~=0.9"
-pytest = "*"
+# TODO: Newer version of mypy above 0.971 such as 0.991 has included new checks such as 'call-overload', 'override', and 'arg-type'
+#       that would introduce new errors during the checks. Set this as a later enhancement for now
```

Comment:
Create issue for this? In case a CVE comes, and we upgrade things might start breaking.

Reply:
Added now:
* https://github.com/opensearch-project/opensearch-build/issues/3704